Framework-User-Guide.md

```diff
@@ -0,0 +1,90 @@
+![logo](https://github.com/culturegraph/metafacture-core/wiki/img/metafacture_small.png)
+
+# Framework User Guide
+
+> [!NOTE]
+>Relevant for JAVA developers. For using metafacture without JAVA Code see the [FLUX user guide](/Flux-User-Guide.md).
+
+This page explains how to create a Metafacture objects and how to assemble them to form a processing pipeline. We use as an example a simple pipeline containing a Metamorph instance.
+
+
+## Building a Flow
+
+A Flow consists of a data source, an arbitrary number of pipe elements and finally a data sink.
+The individual elements are connected by calling the `setReceiver()` method. The following code snipped shows an example.
+
+```java
+// create necessary objects
+final PicaReader reader = new PicaReader();
+final Metafix metafix = new Metafix("defnition.fix");
+final ListMapWriter writer = new ListMapWriter();
+
+//connect them
+reader.setReceiver(fix).setReceiver(writer);
+
+//start processing
+reader.read(input);
+```
+
+Note that the call `setReceiver()` returns
+its argument, preserving the respective type. Thus the calls can be chained to
+build up a pipeline as shown in the listing. Finally the processing is started
+by calling the respective method on the data source/reader. The method name
+depends on the reader. In the Metamorph project `read()` is used by
```

Comment:
resp. `process()`-  I don't know

Reply:
perhapse @fsteeg  can help

Reply:
Yes, like in the other conversation, should be `process`, see https://github.com/metafacture/metafacture-core/blob/9cf783f91487b1f8637a475ca87db8e4e58122e9/metafacture-framework/src/main/java/org/metafacture/framework/ObjectReceiver.java#L40